beacon-chain/rpc/validator_server.go

```diff
@@ -74,6 +76,15 @@ func (vs *ValidatorServer) WaitForActivation(req *pb.ValidatorActivationRequest,
 	}
 }
 
+// WaitTillSync checks if syncservice status is synched and sends message on stream when the service is synced.
+func (vs *ValidatorServer) WaitTillSync(ctx context.Context,_ *ptypes.Empty) (*pb.SyncedResponse, error){
+	if err := vs.syncService.Status();err==nil{
+		return &pb.SyncedResponse{Synced:true},nil
```

Comment:
we dont need a special proto message for this. just return the error from `Status`. We can then handle the error client side. `if err == nil {break} ` or something

Reply:
maybe i will add the sync error separately

Reply:
i am not able to return primitive types in protobuf